sw/host/opentitanlib/src/transport/hyperdebug/mod.rs

```diff
@@ -607,9 +592,62 @@ impl Flavor for StandardFlavor {
     fn gpio_pin(inner: &Rc<Inner>, pinname: &str) -> Result<Rc<dyn GpioPin>> {
         Ok(Rc::new(gpio::HyperdebugGpioPin::open(inner, pinname)?))
     }
+
+    fn spi_index(inner: &Rc<Inner>, instance: &str) -> Result<(u8, u8)> {
```

Comment:
Does the `CW310Flavor` implementation need implementations of `spi_index` and `i2c_index` that forward to `StandardFlavor`?

Reply:
Yes, it does. Good catch.

Reply:
Done.